src/_guides/libraries/index.md

```diff
@@ -43,10 +43,10 @@ The following table lists the Dart core libraries that work on the
 |-----------------------------------------------+-------------------------------|
 | Library                                       | Notes   |
 |-----------------------------------------------|-------------------------------|
-| [`dart:ffi`][dart-ffi]                  <br> A foreign function interface that lets Dart code use native C APIs. | |
-| [`dart:io`][dart-io]                    <br> File, socket, HTTP, and other I/O support for non-web applications. | |
-| [`dart:isolate`][dart-isolate]          <br> Concurrent programming using isolates: independent workers similar to threads. | |
-| [`dart:mirrors`][dart-mirrors]          <br> Basic reflection with support for introspection and dynamic invocation. | Experimental<br>JIT only (_not_&nbsp;Flutter) |
+| [`dart:ffi`][dart-ffi], [`package:ffi`][package-ffi]<br>A foreign function interface that lets Dart code use native C APIs.<br>`package:ffi` contains utilities incl. support for converting Dart strings and C strings. | |
+| [`dart:io`][dart-io], [`package:io`][package-io]<br>File, socket, HTTP, and other I/O support for non-web applications.<br>`package:io` provides additional utilities. | |
+| [`dart:isolate`][dart-isolate], [`package:isolate`][package-isolate]<br> Concurrent programming using isolates: independent workers similar to threads.<br>`package:isolate` provides additional utilities. | |
+| [`dart:mirrors`][dart-mirrors]<br> Basic reflection with support for introspection and dynamic invocation. | Experimental<br>JIT only (_not_&nbsp;Flutter) |
```

Comment:
This is using terms from https://dart.dev/overview#platform. 

If I made this a link to that, would that make it clearer?

Reply:
It would help me, yes.
Consider writing it as "VM JIT" or "JIT VM" since JIT in that link is defined only in relation to the VM. It's not clear it it means something relative to web compilation or not. It's not clear that it _implies_ VM, only that it means something when applied to the VM.

Reply:
Kathy also called out that more clarification here would be helpful: https://github.com/dart-lang/site-www/issues/2077